GoogleForCreators/web-stories-wp
# Update UI for Border and Corner Radius panels

## Summary

Updates the border and corner radius panels UI [per new design](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories---Post-Stable?node-id=1391%3A118).


## Relevant Technical Choices



## To-do

- [x] Adjust tests



## User-facing changes



## Testing Instructions
- Add an element that supports border and verify the design matches the [new Figma](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories---Post-Stable?node-id=1391%3A118) for border width and border radius both.
- Verify it matches the design for locked and unlocked values -- there is just one field for the locked state and 4 fields for unlocked state.


---



Fixes #5531
